### DISCUSSION TITLE
Decoration/etc items.

### DISCUSSION THREAD

**Discussion by SimpleFoxOfficial** [BEEmod/BEE2-items#4115] [Feature Suggestions]
I think will be interesting to use test elements for decoration from those screenshots
![image](https://user-images.githubusercontent.com/82114463/164465775-44635134-5f15-4514-a441-f8744a7e254f.png)
![image](https://user-images.githubusercontent.com/82114463/164465811-a87afdd6-4dbb-4722-ae4b-b66fafe4ceec.png)
![image](https://user-images.githubusercontent.com/82114463/164465853-c5393e60-7bf2-41e2-b5fc-fbd153221e17.png)
![image](https://user-images.githubusercontent.com/82114463/164466442-03e8d45e-bd42-47cd-b522-1c0cc6d1fce0.png)
You can place portals through this vent

**Comment by slambo311** [comment]
The fans could work as a test element too if they pushed objects and players within a certain distance.

**Reply by SimpleFoxOfficial** [reply]
Yep, and would be cool if you'll can to change not the direction(how in faith plates), but change the height
